sensor_msgs/Image color_img
sensor_msgs/Image dist_img
sensor_msgs/Image height_img
sensor_msgs/Image mask_img

